api/example.com/resource/gpu/nas/v1alpha1/client/client.go

```diff
@@ -99,3 +99,17 @@ func (c *Client) Get() error {
 	*c.nas = *crd
 	return nil
 }
+
+func (c *Client) ListNames() ([]string, error) {
+	nasnames := []string{}
+	nases, err := c.client.NodeAllocationStates(c.nas.Namespace).List(
+		context.TODO(), metav1.ListOptions{})
```

Comment:
Either leave this all on one line or put all arguments on a new line. Also, `nases` is a weird name, let's just call it `list` or `naslist`, e.g.:
```
	naslist, err := c.client.NodeAllocationStates(c.nas.Namespace).List(
		context.TODO(),
		metav1.ListOptions{})
```

Reply:
naslist, ok. Happened to move back to one-line with no newlines for arguments before I saw your comments, got convinced by @bart0sh.